Main Cleanup of Indirect Geometry release notes

sf1919 opened the pull request:
**Description of work.**

The main refactoring the the Indirect Geometry release notes ahead of release of 6.2. Includes:-

- Adding links
- Updating titles and headings
- Re-ordering information
- Adding screenshots
- Spell checking


**To test:**

- Check that the links work
- Are the images visible and clearly illustrate the new feature/improvement?
- Grammar/Spelling check

*There is no associated issue.*


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

sf1919 committed:
Initial tidying of Indirect Geometry release notes

Added links, made text more consistent, tidied headings, reordered notes.

sf1919 committed:
Added ElwinLoad GIF

sf1919 committed:
Minor corrections and major refactoring of release notes

Minor spelling, link & formatting corrections. New links in QENS documentation. Major reorganisation to include new Algorithms section.

DavidFair commented on docs/source/release/v6.2.0/indirect_geometry.rst:
```diff
@@ -5,43 +5,47 @@ Indirect Geometry Changes
+New Features
+------------
+- Fit functions :ref:`ElasticIsoRotDiff <func-ElasticIsoRotDiff>` and :ref:`InelasticIsoRotDiff <func-InelasticIsoRotDiff>` have been made available in the :ref:`ConvFit <convfit>` tab in the :ref:`Inelastic Data Analysis <interface-inelastic-data-analysis>` interface.
+- - **The** :ref:`Elwin <Elwin-iqt-ref>` **tab in** :ref:`Inelastic Data Analysis <interface-inelastic-data-analysis>` **has a new loader which now allows users to add workspaces.**
```
This shouldn't be bold - we tend to put all release notes in at the same emphasis with exceptions for very big ticket items (Python 2 -> 3).

sf1919 committed:
Minor changes

Reordering of new feature list, corrected a spelling mistake, corrected a list formatting issue.

DavidFair reviewed (approved):
LGTM and building locally it rendered correctly

martyngigg commented:
Unrelated test failure.
